addons/io/org.openhab.io.homekit/README.md

```diff
@@ -0,0 +1,124 @@
+  <td>&nbsp;</td>
+  <td>heatingCoolingMode</td>
+  <td>String</td>
+  <td>Indicates the current mode of the device: OFF, AUTO, HEAT, COOL. The string's value must match those defined in the thermostat*Mode properties.</td>
+ </tr>
+ <tr>
+  <td>&nbsp;</td>
+  <td>coolingThreshold</td>
+  <td>Number</td>
+  <td>A threshold, above which the thermostat's cooling action will be activated. This only applies when heatingCoolingMode is COOL</td>
+ </tr>
+ <tr>
+  <td>&nbsp;</td>
+  <td>heatingThreshold</td>
+  <td>Number</td>
+  <td>A threshold, below which the thermostat's heating action will be activated. This only applies when heatingCoolingMode is HEAT</td>
+ </tr>
+ <tr>
+  <td>&nbsp;</td>
+  <td>autoThreshold</td>
+  <td>Number</td>
+  <td>A target temperature that will engage the thermostat's heating and cooling actions as necessary. This only applies when heatingCoolingMode is AUTO</td>
+ </tr>
+</table>
+
+Tags are applied to items using the prefix "homekit:". See the sample below for example items:
+
+```
+Switch KitchenLights "Kitchen Lights" <light> (gKitchen) [ "homekit:Lightbulb" ]
+Dimmer BedroomLights "Bedroom Lights" <light> (gBedroom) [ "homekit:DimmableLightbulb" ]
+Number BedroomTemperature "Bedroom Temperature" (gBedroom) [ "homekit:TemperatureSensor" ]
+Group DownstairsThermostat "Downstairs Thermostat" (gDownstairs) [ "homekit:Thermostat" ]
+Number DownstairsThermostatCurrentTemp "Downstairs Thermostat Current Temperature" (gDownstairsThermostat) [ "homekit:currentTemperature" ]
+Number Number DownstairsThermostatHeatingThreshold "Downstairs Thermostat Heating Threshold" (gDownstairsThermostat) [ "homekit:heatingThreshold" ]
+Number DownstairsThermostatCoolingThreshold "Downstairs Thermostat Cooling Threshold" (gDownstairsThermostat) [ "homekit:coolingThreshold" ]
+Number DownstairsThermostatAutoThreshold "Downstairs Thermostat Auto Threshold" (gDownstairsThermostat) [ "homekit:autoThreshold" ]
+String DownstairsThermostatHeatingCoolingMode "Downstairs Thermostat Heating/Cooling Mode" (gDownstairsThermostat) [ "homekit:heatingCoolingMode" ]
+```
+
+## Additional Notes
+HomeKit allows only a single pairing to be established with the bridge. This pairing is normally shared across devices via iCloud. If you need to establish a new pairing, you'll need to clear the existing pairings. To do this, you can issue the command ```clearHomekitPairings``` from the OSGi console.
+
+HomeKit requires a unique identifier for each accessory advertised by the bridge. This unique identifier is hashed from the Item's name. For that reason, it is important that the name of your Items exposed to HomeKit remain consistent.
+
+If you encounter any issues with the add-on and need support, it may be important to get detailed logs of your device's communication with OpenHAB. In order to get logs from the underlying library used to implement the HomeKit protocol, add the following to your logback.xml:
+```<logger name="com.beowulfe.hap" level="TRACE" />```
```

Comment:
logback.xml does not exist any more - best would be the advice to enable trace logging through the shell like this:
```
openhab> log:set TRACE com.beowulfe.hap
```